### ISSUE TITLE
token in a nested object

### ISSUE THREAD

**Issue by artmees** [sahat/satellizer#261]
Setting the auth_token in a nested object is not working
for example, I have a response similar to the following

```json
{
  "users": {
    "id":"54c7b08f4162646d3c190000",
    "token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9...",
    "type":"users",
    "links":{
       "posts": [ "some", "post", "objects", "...." ]
    }
  }
}
```
this returns an error similar to

```
Error: Expecting a token named "token" but instead got: {"users": .... }
```

is there a simple configuration that could make this happen ? I don't wan't to implement an interceptor/transform for the response just to move the token to the root of the doc...

**Comment by sahat** [comment]
There is no simple configuration for it, but if you have any suggestions on how to handle this case let me know.

The code below is where you are running into the problem:

```js
var token = response.access_token || response.data[config.tokenName];
var tokenName = config.tokenPrefix ? config.tokenPrefix + '_' + config.tokenName : config.tokenName;
```

Where `response.data` is an object that you are sending from the server.

**Comment by artmees** [comment]
Fixed in PR #262
